CRITICAL - 2022-01-31 00:28:48 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 00:30:37 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 00:34:57 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 00:35:09 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 00:38:11 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 00:38:51 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
ERROR - 2022-01-31 00:44:57 --> Table 'bca.users' doesn't exist
CRITICAL - 2022-01-31 00:44:57 --> Table 'bca.users' doesn't exist
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Validation\Rules.php(161): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci2\system\Validation\Validation.php(277): CodeIgniter\Validation\Rules->is_unique('gita.refina1@gm...', 'user_email', Array, NULL)
#6 C:\xampp\htdocs\ci2\system\Validation\Validation.php(151): CodeIgniter\Validation\Validation->processRules('email', 'email', 'gita.refina1@gm...', Array, Array)
#7 C:\xampp\htdocs\ci2\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, 'default')
#8 C:\xampp\htdocs\ci2\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#9 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#10 C:\xampp\htdocs\ci2\app\Controllers\userController.php(186): CodeIgniter\Model->insert(Object(stdClass))
#11 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#12 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#13 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#15 {main}
ERROR - 2022-01-31 00:45:43 --> Table 'bca.users' doesn't exist
CRITICAL - 2022-01-31 00:45:43 --> Table 'bca.users' doesn't exist
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Validation\Rules.php(161): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci2\system\Validation\Validation.php(277): CodeIgniter\Validation\Rules->is_unique('gita.refina1@gm...', 'user_email', Array, NULL)
#6 C:\xampp\htdocs\ci2\system\Validation\Validation.php(151): CodeIgniter\Validation\Validation->processRules('email', 'email', 'gita.refina1@gm...', Array, Array)
#7 C:\xampp\htdocs\ci2\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, 'default')
#8 C:\xampp\htdocs\ci2\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#9 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#10 C:\xampp\htdocs\ci2\app\Controllers\userController.php(185): CodeIgniter\Model->insert(Object(stdClass))
#11 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#12 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#13 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#15 {main}
CRITICAL - 2022-01-31 01:05:21 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 01:07:20 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
ERROR - 2022-01-31 01:15:08 --> Unknown column 'user_id' in 'field list'
CRITICAL - 2022-01-31 01:15:08 --> Unknown column 'user_id' in 'field list'
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\userController.php(131): CodeIgniter\Model->insert(Object(stdClass))
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-01-31 01:15:46 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(131): CodeIgniter\Model->insert(Object(stdClass))
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:16:47 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(131): CodeIgniter\Model->insert(Object(stdClass))
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:20:09 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 01:20:37 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 01:22:22 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 01:26:44 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 01:27:04 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 01:27:31 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 01:28:04 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 01:28:19 --> Array to string conversion
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(120): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 01:28:55 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:29:03 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:29:48 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:30:26 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:31:52 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:32:26 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:33:09 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:34:04 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:34:04 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:34:05 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:34:05 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:34:06 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:34:06 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:34:07 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:34:27 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:34:48 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\userController.php(129): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-31 01:39:45 --> There is no primary key defined when trying to make insert.
#0 C:\xampp\htdocs\ci2\system\Model.php(229): CodeIgniter\Database\Exceptions\DataException::forEmptyPrimaryKey('insert')
#1 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#2 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#5 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#6 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2022-01-31 01:42:14 --> Column 'username' cannot be null
CRITICAL - 2022-01-31 01:42:14 --> Column 'username' cannot be null
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-01-31 01:42:59 --> Column 'username' cannot be null
CRITICAL - 2022-01-31 01:42:59 --> Column 'username' cannot be null
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-01-31 01:47:11 --> Column 'username' cannot be null
CRITICAL - 2022-01-31 01:47:11 --> Column 'username' cannot be null
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-01-31 01:48:50 --> Column 'username' cannot be null
CRITICAL - 2022-01-31 01:48:50 --> Column 'username' cannot be null
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-01-31 01:49:41 --> Column 'username' cannot be null
CRITICAL - 2022-01-31 01:49:41 --> Column 'username' cannot be null
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-01-31 01:51:50 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-01-31 01:51:50 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-01-31 01:52:16 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-01-31 01:52:16 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-01-31 01:52:23 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-01-31 01:52:23 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-01-31 01:52:39 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-01-31 01:52:39 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-01-31 01:52:52 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-01-31 01:52:52 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-01-31 01:53:03 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-01-31 01:53:03 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-01-31 01:57:35 --> Column 'nama' cannot be null
CRITICAL - 2022-01-31 01:57:35 --> Column 'nama' cannot be null
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-01-31 02:00:09 --> Trying to get property 'password' of non-object
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(36): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 36, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->login()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 02:00:12 --> Trying to get property 'password' of non-object
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(36): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 36, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->login()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 02:01:03 --> Trying to get property 'password' of non-object
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(36): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 36, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->login()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 02:15:53 --> Undefined variable: id
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(32): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 32, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->login()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 02:25:54 --> Trying to get property 'password' of non-object
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(36): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 36, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->login()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-31 02:27:25 --> Trying to get property 'password' of non-object
#0 C:\xampp\htdocs\ci2\app\Controllers\userController.php(36): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 36, Array)
#1 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\userController->login()
#2 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\userController))
#3 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
ERROR - 2022-01-31 02:31:41 --> Column 'nama' cannot be null
CRITICAL - 2022-01-31 02:31:41 --> Column 'nama' cannot be null
#0 C:\xampp\htdocs\ci2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ci2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ci2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ci2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci2\app\Controllers\RegisterAPI.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci2\system\CodeIgniter.php(824): App\Controllers\RegisterAPI->register_user()
#9 C:\xampp\htdocs\ci2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterAPI))
#10 C:\xampp\htdocs\ci2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
